calico-cloud_versioned_docs/version-3.17/image-assurance/scanners/cluster-scanner.mdx

```diff
@@ -1,62 +1,96 @@
 ---
-description: Identify vulnerabilities in Kubernetes clusters.
+description: Detect vulnerabilities in a Kubernetes cluster.
 ---
 
-# Enable automatic image scanning inside a cluster
+# Scan all images in a Kubernetes cluster
 
-## Big picture
+Scan all images in a Kubernetes cluster to achieve a continuous clean bill of health and defense in depth.
 
-Scan Kubernetes workloads automatically using {{prodname}} Image Assurance cluster scanner.
+Common use cases for scanning in a Kubernetes cluster are:
 
+- Images may pass scanning during the build phase, but they could contain vulnerabilities days or weeks later
```

Comment:
Our continuous scanning feature catches this gap regardless of the method. If you were to scan an image from a CI / CD pipeline in the online mode, the packages for that image would be uploaded and we would rescan them for vulnerabilities everyday.

Reply:
Given this is true for both methods, I added an item in the Overview here. WDYT?

{{prodname}} Image Assurances helps you identify vulnerabilities in workloads that you deploy to Kubernetes clusters. A vulnerability can be a weakness in an application (for example, a design flaw or an implementation bug) that allows attackers to cause additional harm. With Image Assurance you can:

- Scan an image for vulnerabilities
- Assess the impact of newly-found vulnerabilities and prioritize remediation efforts
- Catch vulnerabilities days or weeks later after images pass scanning during the build phase

Reply:
> A vulnerability can be a weakness in an application (for example, a design flaw or an implementation bug)

I think we should be a bit more selective in our definition. For image scanning we are specifically detecting CVEs, which are publicly disclosed / tracked security flaws in common libraries and packages. 

When you say "application" it makes me think that you're talking about the customer's application. If that is what you're referring to, then this doesn't fit with what we detect. We don't detect security flaws or bugs directly found in the end user's application. Instead we detect known flaws in libraries that are used by the user's application. 

You could argue that the common packages are technically applications (e.g. a binary executable like `curl`). But there are times where a CVE is specifically for a library / code package. And those are not "applications".